@Types/modules.d.ts

```diff
@@ -6,5 +6,11 @@ declare module 'sheetclip' {
         }
     };
 
+    export default value;
+}
+
+declare module 'fast-isnumeric' {
+    const value: (value: any) => boolean;
+
```

Comment:
fast-isnumeric has no typing info -- declaring the module's signature as part of this project instead so TS allows its use